content/docs/en/elements/layouts/grid-layout.md

```diff
@@ -1,34 +1,43 @@
 ---
 title: GridLayout
 apiRef: https://docs.nativescript.org/api-reference/modules/_ui_layouts_grid_layout_
-contributors: [rigor789]
+contributors: [rigor789, ikoevska]
 ---
 
-The GridLayout is a layout that arranges its child elements in a table structure of rows and columns. A cell can contain multiple child elements, they can span over multiple rows and columns, and even overlap each other. The GridLayout has one column and one row by default. To add additional columns and rows, you have to specify column definition items (separated by commas) to the `columns` property and row definition items (separated by commas) to the `rows` property of the GridLayout. The width of a column and the height of a row can be specified as an absolute amount of pixels, as a percentage of the available space or automatically:
+`<GridLayout>` is a layout component that lets you arrange its child elements in a table-like manner. 
 
-- **Absolute**: Fixed size of pixels.
-- **Star (*)**: Takes as much space as available (after filling all auto and fixed sized columns), proportionally divided over all star-sized columns. So `3/7` means the same as `30/70`.
-- **Auto**: Takes as much space as needed by the contained child element(s).
+The grid consists of rows, columns, and cells. A cell can span one or more rows and one or more columns. It can contain multiple child elements which can span over multiple rows and columns, and even overlap each other. 
 
-### Samples
+By default, the `<GridLayout>` has one column and one row. You can add columns and rows by configuring the `columns` and the `rows` property. In these properties, you need to set the number of columns and rows and their width and height. You set the number of columns by listing their widths, separated by a comma. You set the number of rows by listing their heights, separated by a comma.
 
-#### A complex grid layout
+You can set a fixed size for column width and row height or you can create them in a responsive manner. 
+
+* **An absolute number:** Indicates a fixed size.
+* **auto:** Makes the column as wide as its widest child or makes the row as tall as its tallest child.
+* **\*:** Takes as much space as available after filling all auto and fixed size columns or rows.
+
+See [Props](#props) for more information.
+
+### Examples
+
+#### Grid layout with fixed sizing
+
+The following example creates a simple 2-by-2 grid with fixed column widths and row heights.
 
 ```html
-<GridLayout columns="40, auto, *" rows="40, auto, *" backgroundColor="#3c495e">
+<GridLayout columns="150, 150" rows="150, 150">
```

Comment:
Change these to `115,115` to stay consistent with other measures (the image displayed is 230 units wide)